app/templates/editor/level/level-edit-view.jade

```diff
@@ -98,6 +98,10 @@ block header
 
             li(class=anonymous ? "disabled": "")
               a(data-i18n="common.fork")#fork-start-button Fork
+            li
+              a#save-branch Save/Stash Branch
+            li
+              a#load-branch Load/Unstash Branch
```

Comment:
This is admin-only, though I forgot to only render these menu items for admins. Will do that and also add `DNT` to these files, cause they should not be translated.